app/src/main/java/com/teamwss/websoso/ui/storage/model/SortType.kt

```diff
@@ -0,0 +1,6 @@
+package com.teamwss.websoso.ui.storage.model
+
+enum class SortType(val apiValue: String, val displayName: String) {
```

Comment:
c: 뷰에서 데이터를 몰라야 하는 건 아니지만, apiValue 보다는 다른 네이밍을 가지면 좋을 것 같아요! 가령 다른 분들의 코드처럼 이런 식으로?
```suggestion
enum class SortType(val titleEn: String, val titleKr: String) {
```

Reply:
수정하겠습니다앙~